concordium-sdk/src/main/java/com/concordium/sdk/ClientV2.java

```diff
@@ -130,6 +131,19 @@ public Iterator<AccountAddress> getAccountList(final BlockHashInput input) {
         return to(grpcOutput, ClientV2MapperExtensions::to);
     }
 
+    /**
+     * Retrieve the Consensus Info  thatcontains the summary of the current state
+     * of the chain from the perspective of the node.
+     *
+     * @param timeoutMillis Timeout for the request in Milliseconds.
+     * @return Concensus Status ({@link ConsensusStatus})
+     */
+    public ConsensusStatus getConsensusInfo(int timeoutMillis) {
```

Comment:
Should one provide a timeout here? Can't a default value be used?

Reply:
Removed the timeout